ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -220,7 +231,7 @@ export class NodeDetailsContent extends React.Component {
               editedNode={this.state.editedNode}
               savedNode={this.state.editedNode}
               setNodeState={newParams => this.setNodeDataAt("ref.parameters", newParams)}
-              createListField={(param, index) => this. createExpressionListField(
+              createListField={(param, index) => this. createParameterExpressionField(
```

Comment:
there is not necessary space at `this. `

Reply:
:+1: